internal/session/provider.go

```diff
@@ -77,6 +77,8 @@ func NewProvider(configuration schema.SessionConfiguration, certPool *x509.CertP
 func (p *Provider) GetSession(ctx *fasthttp.RequestCtx) (UserSession, error) {
 	store, err := p.sessionHolder.Get(ctx)
 
+	store.GetSessionID()
```

Comment:
I don't understand this line. Is it some kind of side effect? If it is it's worth either renaming it or documenting it with a comment.

Reply:
Unintentional, must have been debugging something. Whoops!

Reply:
np